src/WebJobs.Script.WebHost/Controllers/FunctionsController.cs

```diff
@@ -100,4 +100,33 @@ public FunctionsController(WebScriptHostManager scriptHostManager, WebHookReceiv
             return response;
         }
     }
+    
+    // Custom web hooks for extensions. 
+    public class HookController : ApiController
```

Comment:
Some extensions (notably O365 graph) need a http endpoint, which may translate to 1 or more user-level triggers. We can't do this with receivers.